core/lib/config/src/configs/database.rs

```diff
@@ -143,12 +140,59 @@ impl DBConfig {
         60_000
     }
 
+    pub fn backup_interval(&self) -> Duration {
+        Duration::from_millis(self.backup_interval_ms)
+    }
+}
+
+/// Collection of different database URLs and general PostgreSQL options.
+/// All the entries are optional, since some components may only require a subset of them,
+/// and any component may have overrides.
+#[derive(Debug, Clone)]
+pub struct PostgresConfig {
+    /// URL for the main (sequencer) database.
+    pub master_url: Option<String>,
```

Comment:
how about url::Url?

Reply:
I think it will result in [double parsing](https://github.com/matter-labs/zksync-era/blob/main/core/lib/dal/src/connection/mod.rs#L74) without clear benefits. On the other hand, we'll have to add a new dependency here for that.